task1/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
```

Comment:
the latest aws provider is 5.93 now
btw, make sure you understand the version constraints. https://developer.hashicorp.com/terraform/language/expressions/version-constraints